docker-compose.yml

```diff
@@ -1,6 +1,19 @@
 version: '3'
 
 services:
+    reporter:
+        image: liberoadmin/reviewer-reporter:latest
+    environment:
+        MAILER_AUTH_USER: 
```

Comment:
I see there's not default for existing `MAILER_*` variables in `.env`, so let's keep it that way

Reply:
Ah but isn't this causing the build to fail :/

```
[2020-03-05T15:49:07.253Z] + docker-compose down -v
[2020-03-05T15:49:07.587Z] The Compose file './docker-compose.yml' is invalid because:
[2020-03-05T15:49:07.587Z] Unsupported config option for services.environment: 'MAILER_AUTH_USER'
```

Reply:
That failure was the indentation issue.